app/views/admin/users/edit.html.erb

```diff
@@ -18,18 +18,20 @@
         <%= f.text_field :first_name, label: User.human_attribute_name('first_name'), placeholder: User.human_attribute_name('first_name'), required: true %>
         <%= f.text_field :last_name, label: User.human_attribute_name('last_name'), placeholder: User.human_attribute_name('last_name'), required: true %>
         <%= f.text_field :email, label: User.human_attribute_name('email'), placeholder: User.human_attribute_name('email'), required: true %>
+        <%= f.password_field :password, label: User.human_attribute_name('password'), placeholder: User.human_attribute_name('password') %>
+        <%= f.password_field :password_confirmation, label: User.human_attribute_name('password_confirmation'), placeholder: User.human_attribute_name('password_confirmation') %>
```

Comment:
Can we only display these password fields if devise is being used please? 
It's quite confusing to see them with Auth0 is being used.
(I realise this will affect the user tests to do with passwords, so those test could be made pending until this is done https://www.pivotaltracker.com/story/show/176634132)